gci/models.py

```diff
@@ -1 +1,27 @@
-# Create your models here.
+import json
+from django.db import models
+
+
+class Students(models.Model):
+    id = models.IntegerField(default=0, primary_key=True)
+    display_name = models.CharField(max_length=100, default='')
+    organization_id = models.IntegerField(default=0)
+    organization_name = models.CharField(max_length=100, default='')
+    program_year = models.IntegerField(default=0)
```

Comment:
a `DateTimeField` would be a better idea.

Reply:
Why?  This value is like a series .  We'll want to do `>` and `<` comparisons, but I cant see date comparisons occurring.

We could build a separate structure for GCI programs, because we'll no doubt participate again the future, and we can manually add the start & end dates for each GCI (i.e. Nov 28 - Jan 18?), and date comparisons on them would be sensible.

Then `program_year` here is a foreign key.